kubevirt-cdi/step2.md

```diff
@@ -0,0 +1,68 @@
+#### Introduction on Containerized Data Importer
+
+[CDI](https://github.com/kubevirt/containerized-data-importer) is an utility designed to import Virtual Machine images for use with Kubevirt.
+
+At a high level, a PersistentVolumeClaim (PVC) is created. A custom controller watches for importer specific claims, and when discovered, starts an import process to create a raw image named *disk.img* with the desired content into the associated PVC.
+
+#### Install the CDI
+
+We will first explore each component and install them. In this exercise we create a hostpath provisioner and storage class. Also we will deploy the CDI component using the Operator.
+
+`wget https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/storage-setup.yml
```

Comment:
5 commands in one click, it's difficult for the user to understand what is going on, should be 5 different command blocks explaining what is done in each command.